tools/pg_describe

```diff
@@ -11,6 +11,7 @@ const databaseDescribe = require('../lib/databaseDescribe');
 
 const yargs = require('yargs')
     .usage('Usage: $0 <database name> [options]')
+    .demandCommand(1)
```

Comment:
This tool was failing with `Unknown argument: postgres` when I tried to run `./tools/pg_describe postgres -o database`. I'm not sure when this started failing (new major version of `yargs` at some point?) but this change fixes it.

Reply:
I run it as `pg_describe -o database postgres` because when I was at kindergarten they taught us to always put options before positional arguments 😄

Reply:
Lol, FWIW, that's what I tried originally:

```
./tools/pg_describe -o database postgres
Usage: pg_describe <database name> [options]

Options:
      --version         Show version number                            [boolean]
  -o, --output          Specify a directory to output files to
      --ignore-tables   a list of tables to ignore                       [array]
      --ignore-enums    a list of enums to ignore                        [array]
      --ignore-columns  a list of columns to ignore, formatted like
                        [table].[column]                                 [array]
  -h, --help            Show help                                      [boolean]

Examples:
  pg_describe postgres
  pg_describe userdb -o db_description --ignore-tables a b --ignore-columns
  a.col1 a.col2

Unknown argument: postgres
```

The example I posted here was just the most recent thing from my shell history 😂 

It's curious that that worked for you though - when was the last time you ran it successfully?

Reply:
I ran it successfully this morning, but I was running it in PT so it was using the PT version of `node_modules` 😬